babel.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = function (api) {
     presets: ['@babel/preset-typescript'],
     plugins: [
       '@babel/plugin-proposal-class-properties',
+      '@babel/plugin-proposal-optional-chaining',
```

Comment:
Please add a comment about why we use this plugin even though it's supported in core.